public/scripts/timeline.js

```diff
@@ -1597,6 +1608,21 @@ function isValidDate(dateString)
     return day > 0 && day <= monthLength[month];
 };
 
+function generateMementoURIList(object)
+{
+    var URIM = [];
+
+    $.each(object,function(index,obj){
+        if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+            URIM.push(obj.event_link);
+        }
+    });
+    var download = document.getElementById("downloadMementoURI");
+    download.href = "";
+    var data = URIM.join('\n');

```

Comment:
I think we can remove this unnecessary intermediate variable as it is not being reused anywhere other than in the immediate next line. We can perhaps place `URIM.join('\n')` inline in the `encodeURIComponent()` function below.

Reply:
I moved `URIM.join('\n')` inline and tested the changes, everything is working as it should, thanks!